pyomo/contrib/gdpopt/nlp_solve.py

```diff
@@ -10,130 +10,37 @@
 #  ___________________________________________________________________________
 
 """Functions for solving the nonlinear subproblem."""
-from __future__ import division
-
-from math import fabs
+from contextlib import contextmanager
```

Comment:
Since this file has functions to solve linear subproblems I would suggest to rename it to `solve_subproblem.py` or `solve_fixed_subproblem.py`

Reply:
Sure, good point. I think I'll rename `mip_solve` too then, to match.